docs/reference/cli.rst

```diff
@@ -173,3 +173,15 @@ Example::
 
     This command publishes drafts. You should review drafts before using this
     command, because they will become public.
+
+**********************
+Maintenance and repair
+**********************
+
+.. _fix-mptt:
+
+``fix-mptt``
```

Comment:
``fix-mptt`` is replaced by ``tree`` in 3.1 (I totally overlooked this change!) which has the same role